internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -95,8 +95,8 @@ Resources:
     Type: AWS::CloudFront::OriginAccessControl
     Properties:
       OriginAccessControlConfig:
-          Description: Access control for static s3 origin
-          Name: !Sub 'copilot-${AppName}-${EnvName}-${WorkloadName}-origin-access-control'
+          Description: !Sub 'Access control for static s3 origin for ${AppName}-${EnvName}-${WorkloadName}'
+          Name: copilot-origin-access-control
```

Comment:
Is the origin's name supposed to be unique in an account?

Reply:
name is like a "tag" thing. the real id is the resource ID

Reply:
And i moved the previous content for "Name" to "Description"

Reply:
<img width="678" alt="Screenshot 2023-05-18 at 3 52 31 PM" src="https://github.com/aws/copilot-cli/assets/79273084/fc3d33da-2f2e-42bb-a900-299c6e508a41">
I got this from the console though!

Reply:
Oh that's right. Ok i think the only thing we could do here is to truncate the previous name

Reply:
OK I've tested out and it seems like they allow names like `adf-asdf-` which has a "-" at the end.